bin/fuel-core/src/cli/run.rs

```diff
@@ -158,9 +158,6 @@ pub struct Command {
 
     #[clap(long = "tx-pool-ttl", default_value = "5m", env)]
     pub tx_pool_ttl: humantime::Duration,
-
```

Comment:
Could you also update helm chart in `deployment/charts/templates/fuel-core-deploy.yaml`, please?